client/elasticsearch/v710/client.go

```diff
@@ -186,6 +186,49 @@ func (cli *ESClient) AddDocument(ctx context.Context, indexName, documentID stri
 	return nil
 }
 
+// Msearch allows to execute several search operations in one request.
+// See full documentation at https://www.elastic.co/guide/en/elasticsearch/reference/master/search-multi-search.html.
+func (cli *ESClient) MultiSearch(ctx context.Context, searches []client.Search) ([]byte, error) {
+	var body []byte
+	for _, search := range searches {
+		headerByte, err := json.Marshal(search.Header)
+		if err != nil {
+			return nil, err
+		}
+		body = append(body, headerByte...)
+		body = append(body, []byte("\n")...)
```

Comment:
Just append one byte, no need to add a parameterised single element slice of bytes for a single char.
```suggestion
		body = append(body, byte("\n"))
```
NB. Can't be 100% on my syntax, but I think the above is right. 
Also do the same for the line below too.
PS. I do wonder if there's any merit in investigating `io.Pipe()` to save manipulating byteslices in memory. Not used it before but it might be neat. 🤔

Reply:
yes, I tested it locally and it seems it is working adding new lines. Haven't used io pipe to be honest , but i think it creates a pipe reader and pipe writer and in this scenario not even sure we would gain much from it .

Reply:
btw the above suggestion won't work as it cannot convert a string to byte like this.

Reply:
Sorry for my rusty syntax memory, as discussed elsewhere, it should have read `append(body, byte('\n'))`.